src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -578,6 +584,16 @@ module T = struct
               let%map () = Async.Scheduler.yield () in
               List.fold ts ~init:(acc, pending_coinbase_stack_state)
                 ~f:(fun (acc, pending_coinbase_stack_state) t ->
+                  let fee_payer =
+                    Transaction.fee_payer_pk t.With_status.data
+                  in
+                  ( match Signature_lib.Public_key.decompress fee_payer with
+                  | Some _ ->
+                      ()
+                  | None ->
+                      (* TODO: Enable this check. *)
+                      (*raise (Exit (Invalid_public_key fee_payer))*)
```

Comment:
Don't we have to check other PKs as well? pk from coinbase fee transfer and pk from one of the two fee transfers which is not the signer? They won't be the signer but they could still be invalid, right?
Also, can we create an issue (high priority, I think) to add this back after the testnet?